PyInstaller/utils/osx.py

```diff
@@ -602,9 +602,17 @@ def collect_files_from_framework_bundles(collected_files):
             # versioned directory to appease the code-signing gods...
             info_plist_src_top = src_path.parent.parent.parent / "Resources" / "Info.plist"
             if not info_plist_src_top.is_file():
-                raise SystemError(
-                    f"Could not find Info.plist file for .framework bundle in {info_plist_src} or {info_plist_src_top}!"
-                )
+                # Strictly speaking, a .framework bundle without Info.plist is invalid. However, that did not prevent
+                # PyQt from shipping such Qt .framework bundles up until v5.14.1. So by default, we just complain via
+                # a warning message; if such binaries work in unfrozen python, they should also work in frozen
+                # application. The codesign will refuse to sign the .app bundle (if we are generating one), but there
+                # is nothing we can do about that.
+                framework_dir = src_path.parent.parent.parent
+                if compat.strict_collect_mode:
+                    raise SystemError(f"Could not find Info.plist in {framework_dir}!")
+                else:
+                    logger.warning("Could not find Info.plist in %s!", framework_dir)
```

Comment:
Can we mention that codesigning will fail? Otherwise the user's response is most likely to be _so what?_.

Reply:
Good idea.